getting-started/vs-code-integration/dpl-wizard.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
Unless we want to optimize for the keyword "dpl", I suggest that we use a more SEO-friendly URL, for example, `document-processing-wizard.md`